releasenotes/notes/lo-freqs-circuit-c2ec161873eb4a68.yaml

```diff
@@ -0,0 +1,52 @@
+---
+features:
+  - |
+    The ability to pass qubit and measurement LO frequencies has now been added to circuit jobs at
+    both the job and experiment (individual circuit) level.
+
+    At the job level, the fields ``default_qubit_los`` and ``default_meas_los`` may be used as
+    arguents to ``execute`` for both circuit and pulse jobs. The fields are lists of floats
```

Comment:
```suggestion
    arguments to ``execute`` for both circuit and pulse jobs. The fields are lists of floats
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6167/commits/705ca3c97ae84c6f950f430f1eea576d4eb93e1c.